plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcMetadata.java

```diff
@@ -243,10 +242,6 @@ private JdbcTableHandle flushAttributesAsQuery(ConnectorSession session, JdbcTab
                 return Optional.empty();
             }
 
-            while (columnByName.containsKey(SYNTHETIC_COLUMN_NAME_PREFIX + syntheticNextIdentifier)) {
```

Comment:
I wonder if the synthatic name could maybe be more meaningful for the reader to understand what synthetic column is

Reply:
Good point, although feels not closely related to this PR. Mind filing an issue?